x/stakeibc/keeper/lsm_token_deposit.go

```diff
@@ -0,0 +1,51 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v8/x/stakeibc/types"
+)
+
+func (k Keeper) SetLSMTokenDeposit(ctx sdk.Context, deposit types.LSMTokenDeposit) {
+	// TODO [LSM]
+}
+
+func (k Keeper) RemoveLSMTokenDeposit(ctx sdk.Context, denom string) {
+	// TODO [LSM]
+}
+
+func (k Keeper) GetLSMTokenDeposit(ctx sdk.Context, denom string) (deposit types.LSMTokenDeposit, found bool) {
+	// TODO [LSM]
+	return
+}
+
+func (k Keeper) GetAllLSMTokenDeposit(ctx sdk.Context) []types.LSMTokenDeposit {
+	// TODO [LSM]
+	return []types.LSMTokenDeposit{}
+}
+
+func (k Keeper) AddLSMTokenDeposit(ctx sdk.Context, deposit types.LSMDepositStatus) {
```

Comment:
Debatable if this should be a separate function, why not make this the default behavior for the SetLSMTokenDeposit?  You always want to call Add and never call Set right because a direct call to set in the case when there already is a deposit for this chain+val+denom will overwrite the existing with the second.